apps/dav/lib/CardDAV/SystemAddressbook.php

```diff
@@ -92,7 +92,7 @@ public function getChildren() {
 			// Should never happen because we don't allow anonymous access
 			return [];
 		}
-		if (!$shareEnumeration || !$shareEnumerationGroup && $shareEnumerationPhone) {
+		if (!$shareEnumeration || !($shareEnumerationGroup && $shareEnumerationPhone)) {
```

Comment:
Below function has
```suggestion
		if (!$shareEnumeration || (!$shareEnumerationGroup && $shareEnumerationPhone)) {
```

And it looks more sane/compatible with previous behaviour?